ui/src/main/java/com.github.raulccabreu.redukt.ui/StateListenerLayout.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.raulccabreu.redukt.ui
+
+import com.github.raulccabreu.redukt.Redukt
+import com.github.raulccabreu.redukt.states.StateListener
+
+interface StateListenerLayout<T> : StateListener<T> {
+
+    fun getRedukt(): Redukt<T>
+
+    fun start() {
+        getRedukt().listeners.add(this)
+        onChanged(getRedukt().state)
+    }
```

Comment:
What about move start and stop methods directly to state listener @flaviokreis and @raulccabreu ? Because I can't see a reason to have it as child interface.